types/RootEngine.d.ts

```diff
@@ -0,0 +1,16 @@
+export default RootEngine;
+declare class RootEngine {
+    constructor(config: any, resolveInputControls: any, fireNodeFunction: any);
+    config: any;
```

Comment:
By typing the config in typeBuilders we could also type the config here. This config is basically the output of the other one. So no methods but we would know which properties are there. That way Typescript would throw if a config object not created by new FlumeConfig is created.